docker/mlpack/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:latest
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Asia/Kolkata
+
+RUN apt-get update && apt-get install wget build-essential wget cmake gfortran txt2man pkg-config libbfd-dev julia -y && rm -rf /var/lib/apt/lists/* && apt-get autoremove
+
+WORKDIR /tmp
+
+RUN wget -q https://github.com/Reference-LAPACK/lapack/archive/refs/tags/v3.10.0.tar.gz
```

Comment:
Maybe I'm missing something but could't we install the dependencies from the package manager? Would be easier to maintain.

Also, I would propose to use `ARG` to e.g. specify the version, so I can easily switch between master and the latest stable version.

Reply:
> Maybe I'm missing something but could't we install the dependencies from the package manager? Would be easier to maintain.
> 
> Also, I would propose to use `ARG` to e.g. specify the version, so I can easily switch between master and the latest stable version.

Yes, it is possible to install the dependencies from package manager, just thought of installing all dependencies from source, I'll change them to be installed using package manager. Sure, it would be nice to use ARG for version as it would be easy  to switch I'll also modify it.